docs/index.rst

```diff
@@ -74,6 +74,6 @@ If you're looking for something related to the project itself, it's here.
 
   whats_new
   version_guarantees
+  interactions
   migrating
-  migrating_2
```

Comment:
whats with the removing migrating to 2.0

Reply:
Smh again I have to say I moved migrating 2.0 to migrating And removed migrating to 1.0